src/status_im2/contexts/wallet/account/view.cljs

```diff
@@ -25,7 +25,8 @@
   [{:id :assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}
-   {:id :permissions :label (i18n/label :t/permissions) :accessibility-label :permissions}
+   ;; Disabled for new wallet MVP
+   ;{:id :permissions :label (i18n/label :t/permissions) :accessibility-label :permissions}
```

Comment:
can we just remove this for now instead?

Reply:
also cc @briansztamfater @ulisesmac @mmilad75 @tumanov-alex @smohamedjavid @vkjr - removing permissions tab for the moment as requested from product/design. There is some work to do until this is ready 👍